numpy/testing/utils.py

```diff
@@ -3,10 +3,14 @@
 set of tools
 
 """
+from __future__ import division, absolute_import, print_function
+
 import warnings
 
-warnings.warn("Import from numpy.testing, not numpy.testing.utils",
-              ImportWarning)
+# 2018-04-04, numpy 1.15.0
+warnings.warn("Importing from numpy.testing.utils is deprecated, "
+              "import from numpy.testing instead.",
+              ImportWarning, stacklevel=2)
```

Comment:
@charris - any reasons this doesn't become a DeprecationWarning like the rest of this PR?

Reply:
Looks like an oversight while settling on the final form. I don't recall any details at this point :) Want to make a PR?

Reply:
Possibly changing this could break downstream code that was working around it, but hey, the code should get fixed anyway :)